src/modules/stores/context/context.store.ts

```diff
@@ -141,6 +150,31 @@ export class ContextStore extends Store<ContextModel> {
     this.setPageLayoutState();
   }
 
+  // Innovation Record Schema methods.
+  setIrSchema(data: ContextSchemaType): void {
+    this.state.irSchema = data;
+    this.setState();
+  }
+
+  getIrSchema(): InnovationRecordSchemaInfoType {
+    if (!this.state.irSchema?.schema) {
+      return { id: '', version: 0, schema: { sections: [] } };
+    }
+    return this.state.irSchema?.schema;
+  }
```

Comment:
Shouldn't we always use the `getOrLoadIrSchema`?

Reply:
The schema context update follows the same logic as previously used on the 'innovation' context. The `getOrLoadIrSchema` is called on the resolver to populate context, and then `getIrSchema` is used elsewhere to get the current info